config/environments/production.rb

```diff
@@ -76,7 +76,7 @@
   # config.action_controller.asset_host = 'http://assets.example.com'
 
   # Disable e-mails if environment variable is set
-  config.action_mailer.perform_deliveries = ENV['DISABLE_EMAILS'].nil?
+  config.action_mailer.perform_deliveries = ENV['DISABLE_EMAILS']
```

Comment:
This is an interesting case. I'm pretty sure we could use a boolean here but the helper won't be loaded yet (I don't think). That said, we should have similar parsing... I don't know what the best way to handle this is. Same with `RAILS_SERVE_STATIC_FILES` above.